<launch>
  

  <node pkg="rins" type="faces.py" name="facemapper" output="screen">
    <param name="faces_topic" type="string" value="/facedetector/faces" />
    <param name="camera_topic" type="string" value="/camera/rgb/camera_info" />
  </node>
  <node pkg="rins" type="recognizer.py" name="recognizer"/>
  <node pkg="rins" type="cyllinder_detector.py" name="cyllinderdetector" output="screen">
    <param name="camera_topic" type="string" value="/camera/rgb/camera_info" />
  </node>
  <node pkg="rins" type="costmap_approach.py" name="approach" output="screen"/>  


</launch>
